src/main/java/me/gong/mcleaks/MCLeaksAPI.java

```diff
@@ -148,6 +150,7 @@ public String toString() {
         private TimeUnit unit = TimeUnit.MINUTES;
         private boolean testing, noCache;
         private String userAgent = "MCLeaksApiClient", apiKey;
+        private OkHttpClient okHttpClient;

```

Comment:
Initialize the client here instead of in the build method

Reply:
The only problem I see doing it that way is when users create their own instance of the client itself.  If the user creates their own instance, there would probably be two clients because the null check was removed (checks if it was already initialized or not)

Reply:
Alright, you make a good point. Having the initialization in the builder is fine then.